web-console/src/dialogs/overlord-dynamic-config-dialog/__snapshots__/overload-dynamic-config-dialog.spec.tsx.snap

```diff
@@ -11,7 +11,7 @@ exports[`OverlordDynamicConfigDialog matches snapshot 1`] = `
     Edit the overlord dynamic configuration on the fly. For more information please refer to the
      
     <Memo(ExternalLink)
-      href="https://druid.apache.org/docs/0.23.0/configuration/index.html#overlord-dynamic-configuration"
+      href="https://druid.apache.org/docs/latest/configuration/index.html#overlord-dynamic-configuration"
```

Comment:
what made this part change? Shouldn't the console link to the docs of the version of the build? Or is this some artifact of tests and in a release build it would point to the docs version of the release?

Reply:
I changed the doc version to "latest". I think that is a state that makes sense or that setting on the `master` branch. I was imagining that we would cut the release branch and then change the version there. That way there is less "thrashing" on master.

Reply:
Yeah, that makes sense, where is it set at? I looked around but you've got a lot of changes here and couldn't find it 😜

Reply:
It is set on this line: https://github.com/apache/druid/blob/master/web-console/src/links.ts#L22